symfony/src/Controller/NoticeController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\NoticeType;
+use App\Repository\NoticeRepository;
+use App\Service\LoggerService;
+use Doctrine\ORM\EntityManagerInterface;
+use Pmi\Audit\Log;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Form\FormError;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/s/admin/notices")
+ */
```

Comment:
Need to require ROLE_ADMIN for this controller. Or add all of `/admin` to `access_control` in `security.yaml`.

Reply:
Do we have opinions on using `FrameworkExtraBundle` so it can be an annotation, or would we rather explicitly call it in PHP?

Reply:
My opinion is to either use annotations or do pattern matching in the `security.access_control` configuration. Since Silex is currently doing pattern matching, my initial thought is to do that for now, but I'm open to other preferences.

Reply:
I'd agree. We may want the annotations for later work where the permission structure is more complex (and to keep the code centralized), but in the name of keeping this PR simple, I'll update the YAML file.

Reply:
Fixed in 3059fff. Also migrated the error page(s) over. Symfony lets you split them out by code, but I used the fallback template for now.